plugins/autolink/plugin.js

```diff
@@ -9,11 +9,14 @@
 	var doubleQuoteRegex = /"/g;
```

Comment:
This variable is only used in one place, so I don't see a reason why it needs to be detached from the scope it's necessary.

Reply:
Since the regular expression isn't complicated here, I removed variable: https://github.com/ckeditor/ckeditor4/pull/2518/commits/1e6a75833fa6a310f880161d14028617839641c6